plugins/frontend/backstage-plugin-cloudsmith/src/api/CloudsmithClient.ts

```diff
@@ -53,4 +54,56 @@ export class CloudsmithClient implements CloudsmithApi {
     }
     return await response.json();
   }
+
+  // get data from Cloudsmith quota endpoint (https://help.cloudsmith.io/reference/quota_read)
+  async getQuota({ owner }: { owner: string }): Promise<any> {
+    const cloudsmithApiUrl = await this.getApiUrl();
```

Comment:
This can be done once and cached instead of retrieving it each time. Consider adding a variable on the response on line 35 and returning it if its already set.